src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -108,6 +131,13 @@ const SearchAndFilter = ({
     }
   }, [searchData]);
 
+  // Add search prompt value to search on Enter key (code: 32) press
+  const searchPromptKeyPress = (e) => {
+    if (e.charCode === 32) {
```

Comment:
Can you use `e.key === Enter`? Saves comment explaining the codes above.

Reply:
I did try this but originally `e.key` here returns an empty string and `e.keyCode` returns `0`? 🧐

Reply:
Probably because the event is deprecated (see below).